pkg/analytics/analytics.go

```diff
@@ -108,7 +108,7 @@ func SetupAnalyticsProvider(ctx context.Context, opts ...Option) error {
 
 	// Create a new resource with auto-detected host information
 	res, err := resource.New(ctx,
-		resource.WithOS(),
+		resource.WithOSType(),
```

Comment:
@wdbaruni I have reduced the information collected here (from what was requested in the initial POC) to just the operating system type. Prior to this change, this field contained data like:
```
Linux cypress 6.8.0-40-generic #40~22.04.3-Ubuntu SMP PREEMPT_DYNAMIC Tue Jul 30 17:30:19 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
```
Which is beyond the scope of the information we want to share.